draft-demarco-oauth-status-attestations.md

```diff
@@ -322,7 +322,8 @@ When the JWT format is used, the JWT MUST contain the parameters defined in the
 | **iat** | UNIX Timestamp with the time of JWT issuance. | {{RFC9126}}, {{RFC7519}} |
 | **jti** | Unique identifier for the JWT.  | {{RFC7519}} Section 4.1.7 |
 | **credential_format** | The data format of the Credential. Eg: `vc+sd-jwt` for SD-JWT, `vc+mdoc` for ISO/IEC 18013-5 MDOC CBOR [@ISO.18013-5] | this specification |
-| **credential** | It MUST contain the Credential according to the data format given in the `format` claim. | this specification |
+| **credential_hash** | It MUST contain the hash value of the Credential. | this specification |
+| **credential_hash_alg** | The Algorithm used of hashing the Digital Credential. The value SHOULD be set to `S256`. | this specification |
```

Comment:
lets use the IANA named hash registry, and make sha-256 mandatory to implement.

https://www.iana.org/assignments/named-information/named-information.xhtml#hash-alg

Alternatively, we can use the cose registry which includes hash algorithms:

https://www.iana.org/assignments/cose/cose.xhtml#algorithms

the "COSE" style for this kind of thing would be:

```
credential_hash: [alg, hash]
```

Reply:
Regarding the claim name I prefer the first option. 
With regard to the sha-256, I suggest to make the claim OPTIONAL and if omitted we could say that the default value is sha-256. WDYT?

Reply:
I would make it REQUIRED since I love thing explicit. If a day SHA-256 would be vulnerable and another specification woudl extend this one bringin an updated set of algs, the implementation would then always specify the alg making this aspect not ambiguous

Reply:
@OR13 I moved your comment in a [separate issue](https://github.com/peppelinux/draft-demarco-oauth-status-attestations/issues/16), to unblock this PR related to the hash 

A PR wil be create asap, when this PR will be merged

Reply:
See my change suggestion in the form of comment